src/workerd/io/worker.c++

```diff
@@ -3140,6 +3196,15 @@ bool Worker::Isolate::isInspectorEnabled() const {
   return impl->inspector != nullptr;
 }
 
+// =======================================================================================
+
+void Worker::InspectorClient::dispatchProtocolMessage(kj::StringPtr message) {
+  auto& mp = messagePump.getAlreadyLockedExclusive();
```

Comment:
How do we know this is already locked? Can we possibly make the caller pass in `Locked<MessagePump>&` as a parameter to prove it? `getAlreadyLockedExclusive()` is an escape hatch which we should try to avoid most of the time.

Reply:
Done.